src/main/resources/templates/datewarehouse/etl-jobs-config.json

```diff
@@ -187,17 +505,37 @@
 		"sourcePath" : "log.sagebase.org/userGroupSnapshots/",
 		"tableDescriptor": {
 			"name": "usergroupsnapshots",
-			"columns": {
-				"change_timestamp": "timestamp",
-				"snapshot_timestamp": "timestamp",
-				"id": "bigint",
-				"is_individual": "boolean",
-				"created_on": "timestamp"
-
+			"description" : "This table contains snapshots of user-groups. Snapshots are taken when user-groups are created, updated or deleted. Note: Snapshots are also taken periodically and independently of the changes. The snapshot_timestamp records when the snapshot was taken.",
```

Comment:
This table lists all _principals_ (individual users and groups of users).  (A group is the low-level object underlying a team, much like a file handle is the low-level object underlying a file entity.)  In addition to explicit users and teams, principals in Synapse include the _anonymous_ user, the implicit group of all _authenticated_ users, and the implicit _public_ group which includes all users, authenticated or not.